integrations/react/package.json

```diff
@@ -21,24 +21,25 @@
   },
   "scripts": {
     "build": "rimraf dist && npm run build:cjs && npm run build:esm",
-    "build:cjs": "tsc --target es5 --module commonjs --outDir dist/cjs",
-    "build:esm": "tsc --target es6 --module es6 --outDir dist/esm",
-    "serve": "snowpack dev"
+    "build:cjs": "tsc --target es5 --module commonjs --outDir dist/cjs --project tsconfig.build.json",
+    "build:esm": "tsc --target es6 --module es6 --outDir dist/esm --project tsconfig.build.json",
+    "serve": "concurrently vite \"npm run lint\"",
+    "lint": "tsc --project tsconfig.json --watch"
```

Comment:
This will hang in CI since it calls `lint` for each package in the preparation step. Is there any reason the `lint` script is running the TS compiler in watch mode? I'm assuming it's to observe any TS errors but we can drop that since we get feedback immediately in VSCode anyway, and we'll see the error once we run the build.

Reply:
I can't actually see where the lint step is run - the prepare step seems to execute `npm run format` on each package, but only the root (`@vime/core`) package actually defines this command (and aliases it to linting).

The reason to run `tsc` concurrently was to replicate the behaviour of the previous Snowpack configuration (Snowpack's TypeScript plugin basically does exactly that under the hood).

For now, I've renamed the `lint` command to `serve:typecheck` If you think it's better to just remove it and rely on the editor/intellisense instead, let me know and I'll happily take it out 🙂.

Reply:
Looks good thanks!